client/me/concierge/book/is-rebrand-cities-site.js

```diff
@@ -0,0 +1,101 @@
+/** @format */
+
+/**
+ * We have an initiative partnership called Rebrand Cities, where our partners help businesses
+ * set up a WordPress.com Business site. The site owners are then encouraged to book a Concierge
+ * Session to start chatting directly with our Happiness Engineers. We'd like to flag these sessions
+ * as part of the Rebrand Cities program so that the HE has more context and can give the customer
+ * a session tailored to the program.
+ *
+ * The best way to auto-identify a site as belonging to the Rebrand Cities program is by checking
+ * if the main Rebrand Cities WordPress.com account is a member of the site — this account is used
+ * to set up all of these sites.
+ *
+ * This component's purpose is to check the passed-in siteId and notify using the onChange prop
+ * whether the site is a part of the program. This information can then be passed on to the booking
+ * API call and used to customize the invitation and inform the HE that this is a Rebrand Cities session.
+ */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import SiteUsersFetcher from 'components/site-users-fetcher';
+
+const REBRAND_CITIES_ACCOUNT_USERNAME = 'mwtest514';
+
+/**
+ * The <AreUsersPresent /> component takes a set of users and an onChange callback. It
+ * passes true or false into the callback whenever the count of users changes. This is
+ * intended to be wrapped in a SiteUsersFetcher which will pass the users in.
+ *
+ * This is a helper component which isn't exported.
+ */
+class AreUsersPresent extends Component {
```

Comment:
I would say this component is generic enough for any `SiteUsersFetcher` usage. Maybe move it into `components/site-users-fetcher`? Though, `SiteUsersFetcher` is something we ought to get rid eventually so not sure.

Reply:
I'm not certain it's useful enough for other use cases, and to avoid the discussion that comes along with dropping in generic components I'm going to merge as-is. :)